ccd-definition/ComplexTypes/CaseFieldLRspec-CUI-nonprod.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
```

Comment:
I think we only needed one field claimantUserDetails for IDAM not defendant. and also it should be the same as applicantSolicitor1UserDetails fieldtype would be the same as the latter in the cxample so UserDetails (which is a complex type)